apps/studio/src/server/modules/site/site.router.ts

```diff
@@ -136,11 +137,23 @@ export const siteRouter = router({
         userId: ctx.user.id,
         action: "update",
       })
-      if (notificationEnabled) {
-        await setSiteNotification(siteId, notification)
-      } else {
-        await clearSiteNotification(siteId)
-      }
+
+      await db.transaction().execute(async (tx) => {
+        if (notificationEnabled) {
```

Comment:
[question] do we not need to log this?

Reply:
Hmm the logging is done inside `setSiteNotification`, so not explicitly the `notificationEnabled` but the state of the DB change which is from/to undefined for the notification key.